datacatalog/snippets/pom.xml

```diff
@@ -61,5 +65,10 @@
       <version>1.1.3</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.awaitility</groupId>
```

Comment:
please separate tests and not tests dependencies. This one should be higher in the list

Reply:
Moved